Cargo.toml

```diff
@@ -26,7 +25,7 @@ members = [
 "ring" = { git = "https://github.com/briansmith/ring", rev = "3a14ef619559f7d4b69e2286d49c833409eef34a" }
 # Using a local improved version of multiaddr for now
 # TODO: remove once merged upstream
-"multiaddr" = { path = "./rust-multiaddr" }
+"multiaddr" = { git = "https://github.com/multiformats/rust-multiaddr" }
```

Comment:
Why do we use the git version here and 0.2 below?

Reply:
In fact we use `0.2` everywhere, and this line here is an overwrite so that Cargo uses the git version instead of 0.2. The change to `0.2` below is simply a fix.
I've done this while waiting for a version to be released on crates.io.

Reply:
Oh, sorry. I didn't realise it was in a `[patch]` section.